packages/ses/src/compartment.js

```diff
@@ -326,6 +332,18 @@ export const makeCompartmentConstructor = (
       sloppyGlobalsMode: false,
     });
 
+    let evaluator;
+
+    if (hostEvaluators === 'no-direct') {
+      evaluator = () => {
+        throw TypeError(
+          'Compartment evaluation not supported without direct eval.',
+        );
+      };
+    } else {
+      evaluator = safeEvaluate;
+    }
```

Comment:
document this in lockdown.md

Reply:
and not yet visible in userland until `new Compartment()` compatible on Hermes under lockdown (in [tracker](https://github.com/endojs/endo/issues/1891))
`removeUnpermittedIntrinsics` > `intrinsics.%CompartmentPrototype%.importNow.prototype === undefined` > `Uncaught TypeError: property is not configurable` (similar to https://github.com/endojs/endo/pull/2655)